app/models/appeal_submission_upload.rb

```diff
@@ -4,4 +4,8 @@ class AppealSubmissionUpload < ApplicationRecord
   validates :decision_review_evidence_attachment_guid, :appeal_submission_id, presence: true
 
   belongs_to :appeal_submission
+
+  def form_attachment
```

Comment:
Adding this for convenience in https://github.com/department-of-veterans-affairs/vets-api/pull/13973/files#diff-c381f2d647600d39b1f7c499f2dc7fe4e469b30bbad696a1b511531614bbe587R60